### PR TITLE
docker nits

### PR DESCRIPTION
**Pull request by Swiftyos**
### PR Quality Scorecard ✨

- [x] Have you used the PR description template? &ensp; `+2 pts`
- [ ] Is your pull request atomic, focusing on a single change? &ensp; `+5 pts`
- [ ] Have you linked the GitHub issue(s) that this PR addresses? &ensp; `+5 pts`
- [ ] Have you documented your changes clearly and comprehensively? &ensp; `+5 pts`
- [ ] Have you changed or added a feature? &ensp; `-4 pts`
  - [ ] Have you added/updated corresponding documentation? &ensp; `+4 pts`
  - [ ] Have you added/updated corresponding integration tests? &ensp; `+5 pts`
- [ ] Have you changed the behavior of AutoGPT? &ensp; `-5 pts`
  - [ ] Have you also run `agbenchmark` to verify that these changes do not regress performance? &ensp; `+10 pts`

### CODE REVIEW

File: rnd/docker-compose.yml [hunk 1]
```diff
@@ -101,14 +100,13 @@ services:
       - REDIS_PASSWORD=password
       - AUTH_ENABLED=false
       - PYRO_HOST=0.0.0.0
-      - AGENTSERVER_HOST=api_srv
-      - PORT=8002
+      - AGENTSERVER_HOST=rest_server
     ports:
-      - "8002:8002"
+      - "8002:8000"
```
Comment (aarushik93) [inline rnd/docker-compose.yml:105]: what's this change?

File: rnd/docker-compose.yml [hunk 2]
```diff
@@ -127,9 +125,8 @@ services:
       - REDIS_PASSWORD=password
       - AUTH_ENABLED=false
       - PYRO_HOST=0.0.0.0
-      - PORT=8001
     ports:
-      - "8001:8001"
+      - "8001:8000"
```
Comment (aarushik93) [inline rnd/docker-compose.yml:129]: same q

**Review by kcze — COMMENTED** [review]
New names look good to me

File: rnd/docker-compose.yml (hunk 1 above)
Comment (Swiftyos) [reply rnd/docker-compose.yml:105]: As @Pwuts pointed out, as we are using containers we don't need to change the PORT for each entry point of the application, we can just map it here

**Review by aarushik93 — APPROVED** [review]

**Pull request merged**